index.js

```diff
@@ -30,17 +30,24 @@ const getState = (key) => {
   return state[key]; // WARNING: mutable
 };
 
+const _prompt = async (prompt, customConf, message) => {
```

Comment:
Feels like an extra level of indirection... can't we roll this into `prompt()` itself?

Reply:
Removed this and done the required changes to prompt method itself